### PR TITLE
Console_Table

### PR DESCRIPTION
**Pull request by harikt**
Hi Paul, 

I have added the pear console table to aura.

I am keeping the licence without modifying the copy right of source .

Is that ok?

Thank you.

### CODE REVIEW

File: test.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
```
Comment (harikt) [inline test.php:1]: this file is intentionally kept for you to see how it works!

**Comment by harikt** [comment]
@pmjones ping pong. > 1 month :-) .

File: src/Table.php [hunk 2]
```diff
@@ -0,0 +1,922 @@
...
+ * @author   Jan Schneider <jan@horde.org>
+ * @license  http://www.debian.org/misc/bsd.license  BSD License (3 Clause)
+ * @link     http://pear.php.net/package/Console_Table
+ */
+
+namespace Aura\Cli;
+
+use Aura\Cli\Stdio\Formatter;
+
+define('CONSOLE_TABLE_HORIZONTAL_RULE', 1);
```
Comment (harikt) [inline src/Table.php:54]: may be move as constants ?

**Comment by pmjones** [comment]
I wonder if this is a candidate for a Bundle package for all sorts of CLI tooling: Aura.CliTools_Bundle, or something like that?

Also, are there other Console-Table packages worth looking at?

**Comment by harikt** [comment]
@pmjones I took from https://github.com/pear/Console_Table . May be then send the PR to them if they like to change to v2 ?

So users who need console table can use it standalone ?

Another console is currently the one moved to https://github.com/thephpleague/climate .

**Comment by harikt** [comment]
Quick docs http://climate.thephpleague.com/terminal-objects/table/ 

Update : 

http://symfony.com/doc/current/components/console/helpers/tablehelper.html

**Pull request closed without merging**